elixir/apps/web/lib/web/live/settings/identity_providers/okta/show.ex

```diff
@@ -173,12 +173,24 @@ defmodule Web.Settings.IdentityProviders.Okta.Show do
 
     <.danger_zone :if={is_nil(@provider.deleted_at)}>
       <:action>
-        <.delete_button
-          data-confirm="Are you sure want to delete this provider along with all related data?"
-          phx-click="delete"
+        <.button_with_confirmation
+          id="delete_identity_provider"
+          style="danger"
+          icon="hero-trash-solid"
+          on_confirm="delete"
         >
+          <:dialog_title>Delete Identity Provider</:dialog_title>
+          <:dialog_content>
+            Are you sure want to delete this provider along with all related data?
```

Comment:
```suggestion
            Are you sure want to delete this provider? This will remove
            <strong>all</strong> Actors and Groups associated with this provider.
```